android/keyple-plugin/android-nfc/src/main/kotlin/org/eclipse/keyple/plugin/android/nfc/smartremoval/AndroidNfcPluginImpl.kt

```diff
@@ -31,10 +37,17 @@ internal object AndroidNfcPluginImpl : AbstractPlugin(AndroidNfcPlugin.PLUGIN_NA
      * singleton @[AndroidNfcReaderImpl]
     </ProxyReader> */
     override fun initNativeReaders(): ConcurrentMap<String, Reader>? {
-        Timber.d("InitNativeReader() add the unique instance of AndroidNfcReaderImpl")
+        Timber.d("InitNativeReader() add the unique instance of AndroidNfcReaderImpl (with Smart Removal reader)")
```

Comment:
The "with Smart Removal reader" value to log is the only difference with org.eclipse.keyple.plugin.android.nfc.pollingremoval.AndroidNfcPluginImpl
Is possible to better factorize the 2 implementations of AndroidNfcPluginImpl